### PR TITLE
Add ImageJ topic

### PR DESCRIPTION
**Pull request by neuromusic**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

ImageJ is a popular open source platform for image visualization and analysis.

### CODE REVIEW

**Comment by Akoneto9** [comment]
Okay

File: topics/imagej/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+topic: imagej
+display_name: ImageJ
+aliases: fiji, fiji-plugin
+
```
Comment (kenyonj) [inline topics/imagej/index.md]:
```suggestion
```

File: topics/imagej/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+url: https://imagej.net/
+github_url: https://github.com/imagej/imagej
+wikipedia_url: https://en.wikipedia.org/wiki/ImageJ
+
```
Comment (kenyonj) [inline topics/imagej/index.md]:
```suggestion
```

File: topics/imagej/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+short_description: ImageJ is an open source image processing program designed for scientific multidimensional images.
+
```
Comment (kenyonj) [inline topics/imagej/index.md]:
```suggestion
```

**Pull request merged**